clang/lib/StaticAnalyzer/Checkers/WebKit/UncountedCallArgsChecker.cpp

```diff
@@ -25,6 +25,11 @@ using namespace ento;
 
 namespace {
 
+bool stringEndsWith(const std::string &str, const std::string &suffix) {
+  auto index = str.rfind(suffix);
+  return index != std::string::npos && str.size() - suffix.size() == index;
+}
```

Comment:
Consider `llvm::StringRef::ends_with()`.

Reply:
Oh, didn't realize we had that. I'll add a FIXME for making `safeGetName` return a `StringRef`.

Reply:
Fixed!